src/main/kotlin/com/cognifide/gradle/aem/common/instance/tail/Log.kt

```diff
@@ -58,41 +59,39 @@ class Log(
 
         private const val LOGS_SEPARATOR = "  "
 
-        private val DATE_TIME_FORMATTER = DateTimeFormatter.ofPattern("dd.MM.yyyy HH:mm:ss.SSS")
-
         private val PRINT_DATE_TIME_FORMATTER = DateTimeFormatter.ofPattern("yyyy-MM-dd HH:mm:ss.SSS")
 
-        fun create(info: LogInfo, logLines: List<String>): Log {
+        fun create(aem: AemExtension, info: LogInfo, logLines: List<String>): Log {
+
             if (logLines.isEmpty() || logLines.first().isBlank()) {
                 throw TailerException("Passed log entry is empty!")
             }
 
             val fullLog = logLines.joinToString("\n")
 
-            when (val result = matchLogLine(logLines.first())) {
+            when (val result = matchLogLine(logLines.first())?.groups) {
                 null -> throw TailerException("Passed text is not a log entry\nPattern:\n$LOG_PATTERN\nText:\n${logLines.first()}")
                 else -> {
-                    val (timestamp, level, source, message) = result.destructured
                     val followingMessageLines = logLines.slice(1 until logLines.size)
                     return Log(
                         info,
                         fullLog,
-                        parseTimestamp(timestamp, info),
-                        level,
-                        source,
-                        listOf(message) + followingMessageLines
+                        parseTimestamp(result.get("timestamp") ?.value ?.trim() ?: "", aem.datePattern.get(), info),
+                        result.get("level")?.value ?: "",
```

Comment:
I am not sure if `?: ""` is ok here and we should throw error or skip such wrong log entries here